Update install.md (#558)
urbit/urbit.org: content/using/install.md

+++

> title = "Urbit Quickstart Guide"

Reviewer: My personal opinion, btw, is that the word "Install" is sort of important, at least it's the first thing I look for when I scope out a project...

Author: I don't feel strongly, so I'm happy to revert.

aliases = ["/docs/getting-started/"]
+++

> This guide covers how to install and set up your own planet, but installation for a star is exactly the same. You will first install the Urbit binary, then you will either boot a comet or a planet.  It's not too difficult, but if you would prefer to have someone run your urbit for you, consider a [hosting provider](#hosting-providers).

Reviewer 2: The first sentence should mention that this guide covers comets as well as planets.

## Installing Urbit

You'll want the Urbit binary installed first.

> ### Windows

Reviewer 2: It seems a bit strange to list Windows first; usually, the order is MacOS, then Linux, then Windows. (Come to think of it, it might be good to mention urbit/urbit#4257 in the Mac section.)

Author: I agree with you in principle, but practically, the windows instructions have to be done first and then the user moves on to the unix instructions, so to preserve the "forward only" goal of the docs, I put Windows first.

Reviewer 3: I think this is a formatting issue. At some point we should have three tabs

Reviewer 2: So maybe MacOS, then Windows, then Linux?

Author: I don't see any real difference here.  Does anyone else feel strongly?

Reviewer 4: While leading with Windows is aesthetically unappealing, I don't think it presents any major problems.

Author: Updated to reference M1 macs.  

I'm inclined to leave the order as it is.

```
(Where `urbit` is the urbit executable downloaded with `wget` prior)

> Now, you either want to boot either a comet or a planet.

Reviewer 2: There should be a new header somewhere around here to indicate the end of the "here are the specific actions you need to download Urbit" section.

> If you just want to try Urbit out, you can use a comet. They disposable, free identities, that are good for people who are new to Urbit and want to check out the network, but they aren't good for building a reputation as a friendly and responsible resident. Their long names (example: `~dasres-ragnep-lislyt-ribpyl--mosnyx-bisdem-nidful-marzod`) make them difficult to remember—and some communities ban comets to prevent spam—but they're still a good way to see what Urbit is about before buying your own planet.

Reviewer 4: ```suggestion
If you just want to try Urbit out, you can use a comet. These are disposable, free identities, that are good for people who are new to Urbit and want to check out the network, but they aren't good for building a reputation as a friendly and responsible resident. Their long names (example: `~dasres-ragnep-lislyt-ribpyl--mosnyx-bisdem-nidful-marzod`) make them difficult to remember—and some communities ban comets to prevent spam—but they're still a good way to see what Urbit is about before buying your own planet.
```

```
(Where `urbit` is the urbit executable downloaded with `wget` prior)

### Choose to boot a comet or a planet

If you just want to try Urbit out, you can use a comet. They disposable, free identities, that are good for people who are new to Urbit and want to check out the network, but they aren't good for building a reputation as a friendly and responsible resident. Their long names (example: `~dasres-ragnep-lislyt-ribpyl--mosnyx-bisdem-nidful-marzod`) make them difficult to remember—and some communities ban comets to prevent spam—but they're still a good way to see what Urbit is about before buying your own planet.

> To start with a comet, continue below. For planet instructions, see further below.

Reviewer 4: ```suggestion
To start with a comet, continue below. For planet instructions, see [further below](#boot-your-planet).
```

~sampel_marzod:dojo>
```

> When your comet is finished booting, you will see either the `~sampel_marzod:dojo>` or `~sampel_marzod:chat-cli/` prompt. If you see `:chat-cli`, press `Ctrl-X` to switch into Dojo (the Urbit command line).

Reviewer 3: Will you ever see the `chat-cli` prompt on boot? In the interest of simplicity we should probably remove the `ctrl-x` stuff from the initial installation

Author: I agree if true.

Author: I checked, and booting to chat cli is a bug that has been fixed.  I have simplified the language here as a result.

~sampel_marzod:dojo>
```

> When your planet is finished booting, you will see either the `~sampel_marzod:dojo>` or `~sampel_marzod:chat-cli/` prompt. If you see `:chat-cli`, press `Ctrl-X` to switch into Dojo (the Urbit command line).

Reviewer 2: It's not a planet. If you want to be more specific than ship, call it a comet.

To exit Urbit, use `Ctrl-D` or enter `|exit` into Dojo.

> To start your planet up again, run the following from your `urbit` directory (note the lack of `-c` argument):

Reviewer 2: Ditto prior comment. This should be "ship" or "comet".

~sampel_marzod:dojo>
```

When your comet is finished booting, you will see either the `~sampel_marzod:dojo>` or `~sampel_marzod:chat-cli/` prompt. If you see `:chat-cli`, press `Ctrl-X` to switch into Dojo (the Urbit command line).

Get your password to log into landscape (the Urbit web interface) by typing:

```sh

> +code

Reviewer 3: and typing return, presumably

Author: We're not telling people to type return anywhere else.  Not sure this extra instruction is needed.

### Run the boot command {#the-dojo}

Navigate to your `urbit` directory.

Open your keyfile (it is just a text file) and copy the key inside.  Paste it into the following command, with `sampel-palnet` replaced by the name of your
planet.

```sh
./urbit -w sampel-palnet -G rAnDoMkEy
```

Or you can copy the keyfile into the current directory and run:

```sh
./urbit -w sampel-palnet -k ./my-planet.key
```

Either command will create a directory called `sampel-palnet/` and begin building your planet. It may take a few minutes.

When your planet is finished booting, you will see either the `~sampel-palnet:dojo>` or `~sampel-palnet:chat-cli/` prompt. If you're seeing `:chat-cli` press `Ctrl-X` to switch into Dojo (the Urbit command line).

> Get your password to log into landscape (the Urbit web interface) by typing:

Reviewer 2: You also need ```+code``` to start a comet, so this should probably be mentioned in the comet section above.
"Landscape" is usually capitalized.

2.  If the port given is 80, simply type `localhost` into your browser's address bar. If the given port is a different number, such as `8080`, you would type `localhost:8080`. You'll be met with a login prompt.
3. Type your password you got from running `+code` above.
4. You're in! Now you can explore apps such as Chat, Publish, and more.

### Join a group

Now that you're on Landscape, join the Urbit Community, a great place for newcomers to ask questions.

Click the 'Groups' button on your Landscape home page, and then click the `Join Group` column. Then, enter `~bitbet-bolbel/urbit-community` into the field and press the `Join Group` button.

From the Urbit Community group you can join a variety of chatrooms, notebooks, and collections.

## Next Steps

Now that you’re up and running, take some time to deepen your understanding by checking out some of our other guides:

- [Host your Ship in the Cloud](@/using/operations/hosting): Follow this guide to set your ship up to run as a cloud server, accessible from anywhere.
- [Read the Operations Manual](@/using/operations/using-your-ship): Become an expert pilot by learning to use your ship.

> - [Learn to Develop on Urbit](@/using/develop): Learn how to contribute to or develop on Urbit.

Reviewer: It's failing 'cause of these. They need to end in .md — they point to the files in our zola directory here, not to the permalink location.

>=
```

> Now that you’re up and running, skip ahead to the Setup guide.

Reviewer 4: ```suggestion
Now that you’re up and running, skip ahead to the [Setup guide](#setup).
```

## Booting a planet {#boot-your-planet}

There are two options for getting started with a planet. You can purchase one on your own and run Urbit yourself, or you can purchase a planet and hosting services through a [hosting provider](#hosting-providers).

> ### Purchase a planet

Reviewer 4: ```suggestion
### Purchase a planet #{purchase-planet}
```

Note that when you buy a planet, you should ensure with the provider that your parent star is operating. If you need to escape a non-operational star, see [Escaping A Sponsor](@/using/operations/using-your-ship.md#escape).

### Hosting Providers {#hosting-providers}

Hosting providers will often sell you a planet and run it for you. This option is very simple but will probably cost a regular fee.

Urbit is designed to be portable. This means that if you sign up for hosting now but later want to leave your hosting provider and run your Urbit yourself, you should be able to work with them to obtain all of your data and boot your planet back up without losing anything.

Using hosting does mean that you're trusting your provider with your data, but so long as you have your planet, you'll always own your identity.

Current hosting providers are:

- [Tlon Corporation](https://tlon.io): Tlon is the creator of Landscape and is primarily responsible for the creation and maintenance of the Urbit project itself.
- [Get an Urbit ID](https://www.geturbitid.com/): Get an Urbit ID was the first hosting provider and is run by a member of the Urbit community.

> ### Choose to run your planet on your computer or on a cloud service

Reviewer 4: ```suggestion
### Choose to run your planet on your computer or on a cloud service #{local-or-cloud}
```

> ### Choose to run your planet on your computer or on a cloud service #{local-or-cloud}

Reviewer 4: ```suggestion
### Choose to run your planet on your computer or on a cloud service
```

1. Connect to [Bridge](https://bridge.urbit.org). (Bridge is like an Urbit wallet)
2. Enter the name of your planet and the associated master ticket if you have one. Click the "Metamask, Mnemonic, Hardware Wallet..." button if you don't have a master ticket.
3. Once you're logged in, click the "OS" option.
4. Click the "Download Arvo Keyfile" button. If this is grayed out, click on "Reset Networking Keys", then click on "Reset Networking Keys" on the following page, validate the transaction using your wallet, and then click "Send Trasaction". Once the transaction is complete, the "Download Arvo Keyfile" button should be available for you to press. You should receive a `.key` file that contains the secret needed to boot your planet. Hold onto this file.

### Run the boot command {#the-dojo}

Navigate to your `urbit` directory.

Open your keyfile (it is just a text file) and copy the key inside.  Paste it into the following command, with `sampel-palnet` replaced by the name of your
planet.

```sh
./urbit -w sampel-palnet -G rAnDoMkEy
```

Or you can copy the keyfile into the current directory and run:

```sh
./urbit -w sampel-palnet -k ./my-planet.key
```

> Either command will create a directory called `sampel-palnet/` and begin building your planet. It may take a few minutes.

Reviewer 4: ```suggestion
Either command will create a directory called `sampel-palnet/` and begin booting your planet. It may take a few minutes.
```

The language we use elsewhere is booting, not building, so I'd recommend we stick with it.

When your planet is finished booting, you will see `~sampel-palnet:dojo>` (Dojo: the Urbit command line).

Get your password to log into landscape (the Urbit web interface) by typing:

```sh
+code
```

To shut down your planet, use `Ctrl-D`.

To start your planet up again, run the following from your `urbit` directory.

```sh
./urbit sampel-palnet
```

Note that `sampel-palnet/` is the path of a folder, which we just created in your `urbit` directory. This folder is called your planet's **pier**, and it holds all of your data.

Never boot multiple instances of your planet at the same time. You can prevent this from happening on accident by only ever keeping a single copy of your pier.

**Important:** once a key has been used to boot a planet onto the network, it cannot be used to boot that planet again later - doing so will cause communication problems with other planets. For this reason you should **delete the keyfile from your machine once your planet has booted successfully**. If you do use the same key twice, you'll need to reset your planet to restore its functionality.

Delete the keyfile with the command below:

```sh
rm ./my-planet.key
```

> ## Setup

Reviewer 4: ```suggestion
## Setup {#setup}
```

The first thing you generally want to do with a new planet is to **mount** it. A planet being mounted means that it can be seen by Linux, allowing files to be shared between it and your planet. To mount your planet, type `|mount %` at the Dojo prompt.

```
|mount %
>=
```

The `>=` output means that a command was successful. Now you can see your planet's files in its directory.

To understand what happened with this command, read more about Urbit’s filesystem (called “clay”) [here](@/docs/tutorials/arvo/clay.md).

> ### Using Landscape

Reviewer 4: ```suggestion
### Using Landscape #{using-landscape}
```

> ### Using Landscape #{using-landscape}

Reviewer 4: ```suggestion
### Using Landscape
```

Landscape is the Urbit web interface, and it's the best way to interact with your planet. To access Landscape:

1. Start your planet. In the boot messages, look for a line that says something like `http: live (insecure, public) on 80`. The number given is the port that your planet is using.
2.  If the port given is 80, simply type `localhost` into your browser's address bar. If the given port is a different number, such as `8080`, you would type `localhost:8080`. You'll be met with a login prompt.
3. Type your password you got from running `+code` above.

> 4. You're in! Now you can explore apps such as Chat, Publish, and more.

Reviewer 4: ```suggestion
4. You're in! Now you can explore groups, join chat channels, share links, and much more.
```

### Join a group

Now that you're on Landscape, join the Urbit Community, a great place for newcomers to ask questions.

> Click the 'Groups' button on your Landscape home page, and then click the `Join Group` column. Then, enter `~bitbet-bolbel/urbit-community` into the field and press the `Join Group` button.

Reviewer 4: ```suggestion
Click the `Join a Group` button, then enter `~bitbet-bolbel/urbit-community` into the field and press the `Join Group` button.
```

From the Urbit Community group you can join a variety of chatrooms, notebooks, and collections.

> ## Next Steps

Reviewer 4: ```suggestion
### Next Steps
```
